.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -6,7 +6,8 @@ body:
   - type: markdown
     attributes:
       value: |
-        Thanks for your interest in SuperDuperDB. We take all feature requests seriously and will endeavour to respond in a timely manner!
+        Thanks for your interest in SuperDuperDB.
+        We take all feature requests seriously and will response as fat as possible.
```

Comment:
```suggestion
        We take all feature requests seriously and will respond as fast as possible.
```

Reply:
Oh, fsck this is not my file at all, I overwrote my changes with the file from sklearn.

I'm going to just revert this, the original template is fine.